tests-upgrade/tests-sdk1-support/examples/OperationsList.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
What is this folder for?

Reply:
Some swager files reference files that exist in the examples folder. If the example folder is missing, the test case generation fails.